src/controllers/update/update.manage.trusts.review.former.bo.controller.ts

```diff
@@ -2,29 +2,50 @@ import { NextFunction, Request, Response } from 'express';
 
 import { logger } from '../../utils/logger';
 import {
+  UPDATE_MANAGE_TRUSTS_ORCHESTRATOR_URL,
   UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_PAGE,
-  UPDATE_MANAGE_TRUSTS_REVIEW_INDIVIDUALS_URL,
   UPDATE_MANAGE_TRUSTS_REVIEW_THE_TRUST_URL,
+  UPDATE_MANAGE_TRUSTS_TELL_US_ABOUT_THE_FORMER_BO_URL,
 } from '../../config';
+import { getApplicationData, setExtraData } from '../../utils/application.data';
+import { saveAndContinue } from '../../utils/save.and.continue';
+import { getTrustInReview, setTrusteesAsReviewed } from '../../utils/update/review_trusts';
+import { Trust } from '../../model/trust.model';
+import { Session } from '@companieshouse/node-session-handler';
+import { TrusteeType } from '../../model/trustee.type.model';
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
+    const appData = getApplicationData(req.session as Session);
+    const trust = getTrustInReview(appData) as Trust;
```

Comment:
Should not need to use "as Trust"? Or should be checking for undefined which means nothing to review?